lib/anemone/page.rb

```diff
@@ -58,8 +58,9 @@ def links
       return @links unless @links.nil?
       @links = []
       return @links if !doc
+      return @links if no_index?
 
-      doc.search("//a[@href]").each do |a|
+      doc.search('//a[@href and not(contains(@rel, "nofollow"))]').each do |a|
```

Comment:
per CT on slack - after i merged! - that we should make this a setting, not an always on.